StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -561,7 +561,7 @@ void StPicoDstMaker::fillTracks()
   {
     StMuTrack* pTrk = (StMuTrack*)mMuDst->primaryTracks(i);
     if (!pTrk) continue;
-    if (pTrk->id() < 0 || pTrk->id() >= 50000)
+    if (pTrk->id() < 0)
```

Comment:
My problem is that I don't understand the intention behind this check at all.  muDst tells us that the track id is within the range of short. Fine. Why do we want to further reject tracks going into picoDst? We can't have too many tracks in the event or what? What is "too many"?